src/compiler/entries/component-bundles.ts

```diff
@@ -52,24 +65,26 @@ export function generateComponentBundles(
   config: d.ValidatedConfig,
   buildCtx: d.BuildCtx
 ): readonly d.ComponentCompilerMeta[][] {
-  const cmps = sortBy(buildCtx.components, (cmp: d.ComponentCompilerMeta) => cmp.dependents.length);
+  const components = sortBy(buildCtx.components, (cmp: d.ComponentCompilerMeta) => cmp.dependents.length);
 
-  const defaultBundles = getDefaultBundles(config, buildCtx, cmps);
-  const usedComponents = computeUsedComponents(config, defaultBundles, cmps);
+  const defaultBundles = getDefaultBundles(config, buildCtx, components);
+  // this is most likely all the components
+  const usedComponents = computeUsedComponents(config, defaultBundles, components);
 
   if (config.devMode) {
-    return cmps
+    // return only components used in the
```

Comment:
I think this trails off....

Reply:
🤦‍♀️

Reply:
At night in the city. The interior of the room, dimly visible through the window. The programmer, illuminated by the green glow of a curved CRT. Frustration visible, exasperation mounting. The cursed file? `src/compiler/entries/components-bundles.ts`. The offending line? 75. Ah yes, line 75.

The keyboard winces in pain. Only components used in the? What is this, documentation or a cruel joke? Used in there? Used in the _what_!? What could it mean, what could it possibly mean?

(hopefuly we can save something from this frustration 😄)